.flowconfig

```diff
@@ -1,8 +1,42 @@
 [ignore]
```

Comment:
I converted this to be the default shipped with React Native 0.34 - if you need to update you can grab it from the react-native repo